src/app/rosetta/lib/block.ml

```diff
@@ -825,26 +848,34 @@ module Specific = struct
         -> (Block_response.t, Errors.t) M.t =
      fun ~env req ->
       let open M.Let_syntax in
+      let logger = env.logger in
       let%bind query = Query.of_partial_identifier req.block_identifier in
       let%bind res = env.gql () in
       let%bind () =
         env.validate_network_choice ~network_identifier:req.network_identifier
           ~gql_response:res
       in
       let coinbase = (res#genesisConstants)#coinbase in
+      [%log debug] "About to dump response" ;
```

Comment:
Do we need all of this logging going forward?

Reply:
this one is bad, yeah